.github/workflows/rust.yml

```
@@ -0,0 +1,45 @@
+    branches: [ "*" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: blacksmith-2vcpu-ubuntu-2404
+
+    steps:
+    - uses: actions/checkout@v4
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)